pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -22,12 +22,16 @@ import (
 	"io"
 	"net"
 	"net/http"
+	ossys "os"
```

Comment:
This alias for the os package has tripped me up multiple times reading this file.  Just a nit, and you don't have to do it, but please consider using 'o' for the receiver parameter on `OpenStack` functions and not aliasing the `os` package.